app/src/main/res/values/strings.xml

```diff
@@ -580,6 +580,7 @@
     <string name="storage_manager_permission_rationale">In Android 11, you need to allow a special storage permission</string>
     <string name="close_app_text">Close App</string>
     <string name="allow_text">Allow</string>
+    <string name="string_mode">\@string/mode</string>
```

Comment:
what is this one being used for?

Reply:
This is generated by IDE automatically. I will modify the file later.

Reply:
Actually, this string is not used in code.